src/java.desktop/share/classes/com/sun/imageio/plugins/common/SimpleCMYKColorSpace.java

```diff
@@ -63,7 +63,7 @@ public boolean equals(Object o) {
     }
 
     public int hashCode() {
-        return theInstance.hashCode();
+        return getClass().hashCode()+7;
```

Comment:
Alternatively, you could do:
```suggestion
        return System.identityHashCode(theInstance);
```
which seems to have been the original intent.

Reply:
I can do that.